src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/HealthCertifiedPerson.swift

```diff
@@ -215,15 +214,19 @@ class HealthCertifiedPerson: Codable, Equatable, Comparable {
 	private var mostRelevantCertificateTimer: Timer?
 
 	private var completeVaccinationProtectionDate: Date? {
-		if let completeBoosterVaccinationProtectionDate = self.completeBoosterVaccinationProtectionDate {
-			return completeBoosterVaccinationProtectionDate
-		} else if let recoveredVaccinatedCertificate = recoveredVaccinationCertificate,
+		if let recoveredVaccinatedCertificate = recoveredVaccinationCertificate,
 		   let vaccinationDateString = recoveredVaccinatedCertificate.vaccinationEntry?.dateOfVaccination {
-			// if recovery date found -> use it
+			// if recovery vaccination date found
 			return ISO8601DateFormatter.justLocalDateFormatter.date(from: vaccinationDateString)
-		} else if let lastVaccination = vaccinationCertificates.filter({ $0.vaccinationEntry?.isLastDoseInASeries ?? false }).max(),
-				  let vaccinationDate = lastVaccination.vaccinationEntry?.localVaccinationDate {
-			// else if last vaccination date -> use it
+		} else if let completeBoosterVaccinationProtectionDate = self.completeBoosterVaccinationProtectionDate {
+			// if booster vaccination date found
+			return completeBoosterVaccinationProtectionDate
+		} else if let lastVaccination = vaccinationCertificates.filter({ $0.vaccinationEntry?.isLastDoseInASeries ?? false &&
+			$0.vaccinationEntry?.ageInDays ?? 0 > 14 }).max(), let vaccinationDate = lastVaccination.vaccinationEntry?.localVaccinationDate {
+			// if series completion vaccination date found with > 14 days
+			return Calendar.autoupdatingCurrent.date(byAdding: .day, value: 15, to: vaccinationDate)
+		} else if let lastVaccination = vaccinationCertificates.filter({ $0.vaccinationEntry?.isLastDoseInASeries ?? false && $0.vaccinationEntry?.ageInDays ?? 0 <= 14 }).max(), let vaccinationDate = lastVaccination.vaccinationEntry?.localVaccinationDate {
```

Comment:
```suggestion
		} else if let lastVaccination = vaccinationCertificates.filter({ $0.vaccinationEntry?.isLastDoseInASeries ?? false &&
			$0.ageInDays ?? 0 > 14 }).max(), let vaccinationDate = lastVaccination.vaccinationEntry?.localVaccinationDate {
			// if series completion vaccination date found with > 14 days
			return Calendar.autoupdatingCurrent.date(byAdding: .day, value: 15, to: vaccinationDate)
		} else if let lastVaccination = vaccinationCertificates.filter({ $0.vaccinationEntry?.isLastDoseInASeries ?? false && $0.ageInDays ?? 0 <= 14 }).max(), let vaccinationDate = lastVaccination.vaccinationEntry?.localVaccinationDate {
```

If we write it like that we don't need the new `ageInDays` property that we introduced in the VaccinationEntry`. When we discussed it I thought the one on the `HealthCertificate` was not available here.